autogen/agentchat/contrib/vectordb/base.py

```diff
@@ -0,0 +1,195 @@
+from typing import Any, Callable, List, Protocol, runtime_checkable, Dict
+
+
+@runtime_checkable
+class VectorDB(Protocol):
+    """
+    Abstract class for vector database. A vector database is responsible for storing and retrieving documents.
+    """
+
+    def __init__(self, db_config: Dict = None) -> None:
+        """
+        Initialize the vector database.
+
+        Args:
+            db_config: Dict | configuration for initializing the vector database. Default is None.
+
+        Returns:
+            None
+        """
+        ...
+
+    def create_collection(self, collection_name: str, overwrite: bool = False, get_or_create: bool = True) -> Any:
```

Comment:
Can we do better than a return type of `Any`?

Reply:
It would be better to return None.

Reply:
But if it can vary from collection to collection, how do we use this in a generic capacity?